tests/hazmat/primitives/test_ec.py

```diff
@@ -181,7 +181,7 @@ def test_encode_point():
         16
     )
     pn = ec.EllipticCurvePublicNumbers(x, y, ec.SECP256R1())
-    with pytest.warns(utils.DeprecatedIn25):
+    with pytest.warns(utils.CryptographyDeprecationWarning):
```

Comment:
Why not use PersistentlyDeprecatedIn2019 here?

Reply:
Uhhh, wtf, I have no idea.